### PR TITLE
Usage example added to tf.image.transpose

### PR DESCRIPTION
**Pull request by Mbah-Javis**
A small usage example added to tf.image.transpose

### CODE REVIEW

**Comment by Mbah-Javis** [comment]
@googlebot I signed it!

File: tensorflow/python/ops/image_ops_impl.py [hunk 1]
```diff
@@ -640,6 +640,11 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example:
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
  Usage Example: 
  ```python
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 2]
```diff
@@ -640,6 +640,11 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example: 
+    import tensorflow as tf
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 3]
```diff
@@ -640,6 +640,11 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example: 
+    import tensorflow as tf 
+    x = tf.random.normal(shape=(256, 256, 3))
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    >>> x = tf.random.normal(shape=(256, 256, 3)) 
```

File: tensorflow/python/ops/image_ops_impl.py [hunk 4]
```diff
@@ -640,6 +640,11 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example: 
+    import tensorflow as tf 
+    x = tf.random.normal(shape=(256, 256, 3)) 
+    tf.image.transpose(x)
```
Comment (kyscg) [inline tensorflow/python/ops/image_ops_impl.py]:
```suggestion
    >>> tf.image.transpose(x)  
    ```
```

**Review by kyscg — CHANGES_REQUESTED** [review]
There's no need to add import tensorflow everywhere, just merge the changes I've made

**Comment by Mbah-Javis** [comment]
Hello I have made the changes

**Comment by alextp** [comment]
I'm closing this PR since I've made the comment N times that you should use a small constant and not a random image and should make it a doctest.

**Comment by Mbah-Javis** [comment]
> I'm closing this PR since I've made the comment N times that you should use a small constant and not a random image and should make it a doctest.

I have done that on this other 
[PR](https://github.com/tensorflow/tensorflow/pull/35345/files)

**Pull request closed without merging**